pkg/translator/azure/resourcelogs_to_logs.go

```diff
@@ -37,7 +38,12 @@ const (
 	azureTenantID          = "azure.tenant.id"
 )
 
-var errMissingTimestamp = errors.New("missing timestamp")
+var (
+	errMissingTimestamp       = errors.New("missing timestamp")
+	errNotSupportedTimeFormat = errors.New("not supported time format")
```

Comment:
```suggestion
	errNotSupportedTimeFormat = errors.New("unsupported time format")
```

Reply:
removed `errNotSupportedTimeFormat` since it's not used.